packages/tracing/src/browser/browsertracing.ts

```diff
@@ -189,6 +198,10 @@ export class BrowserTracing implements Integration {
       registerBackgroundTabDetection();
     }
 
+    if (_experiments?.interactionSampleRate ?? 0 > 0) {
+      this._registerInteractionListener(_experiments?.interactionSampleRate ?? 0);
```

Comment:
Nit but the linter probably complains about this in multiple occurances in this PR:

We don't use optional chaining and nullish coalescing operators in the SDK as their bundle size impact is higher than using "traditional" checks like 

Also, for this particular case: It might be the missing morning coffee but I don't entirely understand the `0 > 0` check. Wouldn't this just evaluate to `false`?

```suggestion
    if (_experiments && _experiments.interactionSampleRate != null) {
      this._registerInteractionListener(_experiments.interactionSampleRate || 0);
```

Reply:
Thanks for looking, both of you 👏 

> optional chaining and nullish coalescing operators in the SDK as their bundle size impac

Right 😄, I do know that but it's been a bit, I forgot. I'll re-arrange this. Too much product code recently 😏 

> not register the listener when interactionSampleRate === 0

The default experiments object that gets set into this should be making this 0, if it's either unset or 0, either way we do not want to add a listener as that would affect the side not on the experiment.

> I wonder if having a dedicated interactionTracesSampleRate is the way to go in the long run

It's definitely not, you're right. I'd prefer we're smarter about which ones we send, always trying to make sure we have the one represented by INP when I add it, and leverage biasing in dynamic sampling. For now I've added this rate for experimenting since I'm concerned about sending many _many_ times the transactions (interactions are way more frequent on an SPA) for our frontend than we currently do. In general I'd be leaning towards simple / no-config defaults but still offering control in some manner (either here or elsewhere, such as DS) if a user chooses to do so.

I can see users just wanting everything to be sent especially if they are smaller and are easily within their transaction quota. Something to consider before we un-experiment this, at least.

Reply:
> added this rate for experimenting since I'm concerned about sending many many times the transactions

For now can we just use `tracesSampler` to sample based on op name in the frontend and remove logic around a dedicated sample rate? The mobile SDK implementations do not have this. Saves us bundle size also.

> I can see users just wanting everything to be sent especially if they are smaller and are easily within their transaction quota

In order to make sure we don't blow up quota we'll have to do the following.

1. Gate this behind some kind of flag (ideally boolean), but default it to false
2. Update the performance monitoring docs `Sentry.init` instructions to have the flag on by default, so new users get this functionality automatically
3. Flip the flag in the major version